### PR TITLE
three -> four extensions

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: extensions/readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Use subfolders under the `extensions` folder to house extensions to the final unified hosts file.
 
-Currently this repo includes three extensions: 
+Currently this repo includes four extensions: 
 
 * `gambling` for common online betting sites,  
 * `social` for common social media sites, and
```
Comment (gfyoung) [inline extensions/readme.md]: I think you should also move the "and" as well, since there are four elements, not three.

**Comment by StevenBlack** [comment]
Thanks @Tobias-B-Besemer good catch.  If you include @gfyoung's suggestion, I'll merge.

File: extensions/readme.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
 
 Use subfolders under the `extensions` folder to house extensions to the final unified hosts file.
 
-Currently this repo includes three extensions: 
+Currently this repo includes four extensions:
```
Comment (gfyoung) [inline extensions/readme.md]: Nit: "Current this" --> "Currently, this"

**Comment by ScriptTiger** [comment]
Oh, come on, let the poor guy have his PR. If you want to do a line edit, open another one. @Tobias-B-Besemer, you are German, correct? So English is not your first language? lol

**Comment by Tobias-B-Besemer** [comment]
Yes, I'm a German. ^^

**Comment by gfyoung** [comment]
> Oh, come on, let the poor guy have his PR. If you want to do a line edit, open another one. 

Hey, cut me some slack.  I divide my time equally between contributor and maintainer, so apologies if one side mixes with the other :smile:  That being said, I don't think I'm asking too much here :wink:

**Comment by Tobias-B-Besemer** [comment]
No prob! ;-)

**Comment by StevenBlack** [comment]
Danke Tobias @Tobias-B-Besemer !

**Pull request merged**